ignite/services/plugin/plugin.go

```diff
@@ -142,8 +142,19 @@ func newPlugin(pluginsDir string, cp pluginsconfig.Plugin, options ...Option) *P
 		apply(p)
 	}
 
-	if strings.HasPrefix(pluginPath, "/") {
-		// This is a local plugin, check if the file exists
+	// This is a local plugin, check if the file exists
+	if tp := (&Plugin{Plugin: pluginsconfig.Plugin{Path: pluginPath}}); tp.IsLocalPath() {
```

Comment:
IMHO, we can use only a helper method  directly instead of instantiating a new Plugin pointer to use only one method

Reply:
This was for avoiding logic duplication.l, but I can duplicate